components/validator/l2_output_submitter.go

```diff
@@ -251,25 +243,25 @@ func (l *L2OutputSubmitter) CalculateWaitTime(ctx context.Context, nextBlockNumb
 	return 0
 }
 
-// hasEnoughDeposit checks if validator has enough deposit to bond when trying output submission.
-func (l *L2OutputSubmitter) hasEnoughDeposit(ctx context.Context) (bool, error) {
+// HasEnoughDeposit checks if validator has enough deposit to bond when trying output submission.
+func (l *L2OutputSubmitter) HasEnoughDeposit(ctx context.Context) (bool, error) {
```

Comment:
I just wonder why you make this function public.

Reply:
This function is called in `CalculateWaitTime`, and `CalculateWaitTime` is called in test code. If a function using `ctx` is used in test code, we need to receive `ctx` as an argument. I wanted to mark a function using `ctx` and used in test code as a public function.